src/modules/fetchScores.js

```diff
@@ -0,0 +1,33 @@
+import Storage from './storage.js';
+
+export default class FetchScores {
+  static getScore(url) {
+    fetch(url)
+      .then((response) => response.json())
+      .then((json) => {
```

Comment:
- [ ] **Required:** Your app works correctly. Kudos. 👏 
- However, you're required to use arrow functions and async, await calls to post to and fetch data from the API. Kindly update your functions accordingly.

![image](https://user-images.githubusercontent.com/27709832/167851217-202e795a-f3f8-40ac-8a63-8ead03fa3a0c.png)